.github/workflows/cmake.yml

```diff
@@ -0,0 +1,158 @@
+name: CMake
+
+on: [push, pull_request]
+  
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions/managing-complex-workflows#using-a-build-matrix
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [ubuntu-18.04, windows-2019, macos-10.15]
+        include:
+          - os: ubuntu-18.04
+            c-compiler: "gcc"
+            cxx-compiler: "g++"
+            itk-git-tag: "v5.1.1"
+            cmake-build-type: "MinSizeRel"
```

Comment:
@ViktorvdValk Is there a specific reason for "MinSizeRel", instead of simply "Release"?  As we do at Azure Pipelines: https://github.com/SuperElastix/elastix/blob/5.0.1/Testing/CI/Azure/ci.yml#L124